importers/djornl/parser.py

```diff
@@ -646,22 +666,18 @@ def load_data(self, dry_run=False):
             if output['err_list']:
                 all_errs = all_errs + output['err_list']
 
-        if all_errs:
-            raise RuntimeError("\n".join(all_errs))
+        # save the dataset, unless this is a dry run
+        if not dry_run:
+            self.save_dataset()
```

Comment:
This is a change in functionality. The parser should die with a list of errors and _not_ upload files if there are any problems with the input; there's another ticket (#57) to add a `force` option to it that will load data even if there have been errors. I want to be able to use the parser to QC data, so having it die if it encounters any problems is important.